ferry.go

```diff
@@ -372,14 +421,24 @@ func (f *Ferry) RunStandaloneDataCopy(tables []*schema.Table) error {
 		return nil
 	}
 
-	dataIterator, err := f.newDataIterator()
+	dataIterator, err := f.newDataIterator(false)
 	if err != nil {
 		return err
 	}
 
+	// HACK: This process is not interruptible, so we can override the previous
+	// StateTracker for this instance only as newDataIterator uses the Ferry's
+	// instance of StateTracker.
+	// BUG: if the PanicErrorHandler fires while running the standalone copy, we
+	// will get an error dump even though we should not get one, which could be
+	// misleading.
+	// TODO: refactor this to make it better, possibly have to refactor
+	// newDataIterator.
+	dataIterator.StateTracker = NewStateTracker(f.DataIterationConcurrency * 10)
```

Comment:
I don't like this line, but to do it properly would involve having to refactor newDataIterator and this method in general. This is a yakshave I don't want to undertake right now.

When we do refactor this, we should make the DataIterator/BinlogStreamer/BinlogWriter/BatchWriter easier to use as they're very difficult to initialize independently of Ferry (which is why we even have newDataIterator...).